src/Evolution/Systems/NewtonianEuler/BoundaryCorrections/Hll.hpp

```diff
@@ -0,0 +1,179 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+#include <optional>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/NewtonianEuler/BoundaryCorrections/BoundaryCorrection.hpp"
+#include "Evolution/Systems/NewtonianEuler/Tags.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Formulation.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace NewtonianEuler::BoundaryCorrections {
+/*!
+ * \brief An HLL (Harten-Lax-van Leer) Riemann solver for NewtonianEuler system
+ *
+ * Let \f$U\f$ be the evolved variable, \f$F^i\f$ the flux, and \f$n_i\f$ be the
+ * outward directed unit normal to the interface. Denoting \f$F := n_i F^i\f$,
+ * the HLL boundary correction is \cite Harten1983
+ *
+ * \f{align*} G_\text{HLL} = \frac{\lambda_\text{max} F_\text{int} +
```

Comment:
we usually put the `\f{align*}` and `\f}` on its own line. This makes the C++ comment easier to read (and Doxygen parsing can be annoying sometimes, leading to errors)

Reply:
Okay!